testing/helpers.ts

```diff
@@ -6,6 +6,7 @@ import * as readlineSync from 'readline-sync';
 import {translateErrorStackFromVM} from '../runtime/common/source_map';
 import {unwrapError} from '../runtime/common/marshaling';
 import {wrapError} from '../runtime/common/marshaling';
+import yn from 'yn';
```

Comment:
How much do we care about using this vs rolling our own simple heuristic? Based on yarn.lock, this is pulling in many new packages, just for some simple string checks, not sure if it's worth it.

Reply:
I don't think `yn` has an dependencies:

https://github.com/sindresorhus/yn/blob/main/package.json

I think the ones you are seeing in `yarn.lock` are from `proxyquire`.